### DISCUSSION TITLE
打了deb包，有需要自取。

### DISCUSSION THREAD

**Discussion by wcbing** [nxtrace/NTrace-core#277] [General]
命名有点乱，社区、组织、程序名都是 NextTrace，但是组织的用户名和官网是 nxtrace，这我还可以认为是简称，但是为什么主程序的仓库名又叫 [NTrace](https://github.com/nxtrace/NTrace-core)，这都已经三个名字了😂

---

另外，为了我个人更新方便，我打了[deb包](https://github.com/wcbing-build/nexttrace-debs)，有需要自取。

**Comment by tsosunchia** [comment]
因为好的名字已经被注册了

**Reply by wcbing** [reply]
那好奇为什么又出现一个 [NTrace](https://github.com/nxtrace/NTrace-core) （n后没x）

**Reply by tsosunchia** [reply]
单纯是因为 ntrace被注册了

**Reply by tsosunchia** [reply]
理想的情况是

github.com/ntrace
作为org仓库

ntrace.org
作为官方网站

github.com/ntrace/ntrace-core
作为nexttrace主仓库


可惜没有如果

**Comment by tsosunchia** [comment]
非常感谢你帮忙做了个deb仓库，建议你再pr个readme补丁，把此作为一个安装方式加入介绍

**Reply by wcbing** [reply]
交到哪个仓库😳

**Reply by tsosunchia** [reply]
ntrace-v1 仓库

**Reply by tsosunchia** [reply]
https://github.com/nxtrace/NTrace-V1

**Comment by wcbing** [comment]
@tsosunchia 我看到你有 [fork](https://github.com/nxtrace/nexttrace-debs) 一份，如果想官方维护那肯定是最好的。因为主仓库中支持的架构比较多，所以 debs 还是单独一个仓库？我可以把这个仓库转给你。

然后说一下现在的触发方式，是通过 cron 触发的，每天一次。这样肯定有延迟，也造成一定的资源浪费，如果你接手后有两种改进方法。
- 一是改为创建tag触发，需要每次由你推送主仓库后再到debs仓库手动触发。
- 二是修改主仓库，加一个 workflow，通过 workflow-dispatche 自动触发。我做了个例子可以看一下。
    - [用于主仓库的 workflow](https://github.com/wcbing/NTrace-core/actions/workflows/triggerDebRepo.yml)
    - [debs 仓库](https://github.com/wcbing/nexttrace-debs)

**Reply by tsosunchia** [reply]
https://github.com/nxtrace/NTrace-core/pull/290

**Reply by wcbing** [reply]
好的，应该可以正常触发吧。

**Reply by tsosunchia** [reply]
还不确定，等下次发版了

**Reply by wcbing** [reply]
我之前已经把你加为[原来仓库](https://github.com/wcbing-build/nexttrace-debs)的管理员了，你可以将其转移到你组织下面了。

**Reply by tsosunchia** [reply]
> 我之前已经把你加为[原来仓库](https://github.com/wcbing-build/nexttrace-debs)的管理员了，你可以将其转移到你组织下面了。

okay

**Reply by wcbing** [reply]
有一点问题，能操作 nexttrace-debs 仓库的 token 应该放进 NTrace-core 仓库的 secrets，因为它是触发后发出请求的一方。
其他的应该没问题了。

**Reply by tsosunchia** [reply]
> 有一点问题，能操作 nexttrace-debs 仓库的 token 应该放进 NTrace-core 仓库的 secrets，因为它是触发后发出请求的一方。 其他的应该没问题了。

两个仓库我都加了
